source/macos/AdaptiveCards/AdaptiveCardsTests/Renderers/InputDateRendererTests.swift

```diff
@@ -50,6 +50,13 @@ class InputDateRendererTest: XCTestCase {
         XCTAssertEqual(inputDateField.maxDateValue, maxValue)
     }
     
+    func testRendererForIsRequired() {
+        inputDate = .make(isRequired: true)
+        
+        let inputDateField = renderDateInput()
+        XCTAssertTrue(inputDateField.isRequired)
+    }
+    
```

Comment:
Will you be adding a test to check for isValid getting set right?

Reply:
You mean a test for what isValid returns in case of valid and invalid inputs, right?

Reply:
Yepp
That will be a good check for regression later on in all inputs